.github/workflows/ci-build.yml

```diff
@@ -225,25 +240,34 @@ jobs:
     - name: Checkout
       uses: actions/checkout@v2
 
-    # "restore-cache" which is done by "run-vcpkg" uses Windows tar.
-    # A git clone on windows marks a few files as read-only; when Windows tar
-    # tries to extract the cache over this folder, it fails, despite the files
-    # being identical. This failure shows up as an warning in the logs. We
-    # avoid this by simply removing the read-only mark from the git folder.
-    # In other words: this is a hack!
-    # See: https://github.com/lukka/run-vcpkg/issues/61
-    - name: Remove read-only flag from vcpkg git folder
+    - name: Prepare cache key
+      id: key
       shell: powershell
       run: |
-        attrib -r "c:\vcpkg\.git\*.*" /s
+        # Work around caching failure with GNU tar
+        New-Item -Type Junction -Path vcpkg -Target c:\vcpkg
 
-    - name: Prepare vcpkg (with cache)
-      uses: lukka/run-vcpkg@v6
+        cd c:\vcpkg
+        $commitid = git show -s --format=%H
+        Write-Output "::set-output name=commitid::$commitid"
```

Comment:
If you like, you can also use `bash` as shell, allowing you to re-use what we did for macOS, I believe?

Reply:
I use powershell syntax to create the NTFS junction (the workaround), not sure it's possible with bash.